tests/unit/FormCheckbox.spec.js

```diff
@@ -0,0 +1,81 @@
+import { shallowMount } from '@vue/test-utils'
+import FormCheckbox from '../../src/components/FormCheckbox.vue';
+
+describe('FormCheckbox',() => {
+    let wrapper;
+    const label = 'Accept Terms';
+    beforeEach(() => {
+        wrapper = shallowMount(FormCheckbox, {
+            propsData: {
+                label: label
+            }
+        });
+    });
+
+    it('renders the component', () => {
+        expect(wrapper.find('input').attributes().type).toBe('checkbox');
+        expect(wrapper.html()).toContain(label);
+    });
```

Comment:
Considering the other tests already in this file, does this tests still add any value? Let's say we refactor the component to use a custom checkbox, or a different input type, without breaking any of the actual user-facing functionality, this test will fail even though the component may actually be working correctly.